tests/fireproof/repro-meta-refs-direct.test.ts

```
@@ -0,0 +1,174 @@
+      const blockstore = db.ledger.crdt.blockstore;
+      
+      console.log("=== CarLog State Transitions ===");
+      
+      // Track carLog changes
+      const logCarLogState = (label: string) => {
+        const carLog = blockstore.loader.carLog.asArray();
+        console.log(`${label}: ${carLog.length} entries - ${carLog.map(g => g.map(c => c.toString().slice(-8)).join(',')).join(' | ')}`);
+        return carLog;
+      };
+
+      const initial = logCarLogState("Initial");
```

ESLint

[error] 'initial' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)